tests/cephfs/cephfs_scale/max_smallfile_snap.py

```diff
@@ -6,18 +6,16 @@
 from ceph.ceph import CommandFailed
 from tests.cephfs.cephfs_utilsV1 import FsUtils
 from utility.log import Log
+from utility.utils import convert_bytes, get_storage_stats
 
 log = Log(__name__)
 
 
-def get_available_space(client, fs_name="cephfs"):
-    out, rc = client.exec_command(
-        sudo=True, cmd=f"ceph fs status {fs_name} --format json"
-    )
-    output = json.loads(out)
-    return next(
-        (pool["avail"] for pool in output["pools"] if pool["type"] == "data"), None
-    )
+def get_dd_for_each_iteration(client, pool_name):
```

Comment:
what does dd mean ?

Reply:
for each iteration, i wanted to get how many mb data needs to be filled. so i named it as dd.
Changing it to get_data_to_fill_for_each_iteration

Reply:
Okay